testData/solutions/win32.json5

```diff
@@ -7556,6 +7563,70 @@
                 "inverseCapabilitiesTransformations": {
                     "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.MouseSonar": "MouseSonarConfig.value"
                 }
+            },
+            "configureDoubleClickWidth": {
+                "type": "gpii.windows.nativeSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "functionName": "DoubleClickWidth"
+                },
+                "supportedSettings": {
+                    "DoubleClickWidthConfig": {
+                        "schema": {
+                            "title": "Double-click rectangle width",
+                            "description": "Changes the width of the rectangle in which a second click should fall for OS to consider the mouse action a double-click.",
```

Comment:
"... should fall to be considered as a double-click action"  sounds better to me.

Reply:
Forgot about this one, doing it.

Reply:
Oh, I see, then leave it as it is.